worth2/main/generic/models.py

```diff
@@ -78,7 +78,10 @@ def percent_complete_hierarchy(self):
             return 0
 
     def time_spent(self):
```

Comment:
No, this model was copied over from MeaningfulConsent -- https://github.com/ccnmtl/meaningfulconsent/blob/master/meaningfulconsent/main/models.py#L99

Maybe we can remove this function? I figured that eventually we might want to move this model into django-pagetree.